components/content/inspira/ui/text-reveal-card/TextRevealCard.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div
+    @mouseenter="mouseEnterHandler"
+    @mouseleave="mouseLeaveHandler"
+    @mousemove="mouseMoveHandler"
+    @touchstart="mouseEnterHandler"
+    @touchend="mouseLeaveHandler"
+    @touchmove="touchMoveHandler"
+    ref="cardRef"
+    :class="[
+      'w-full max-w-[40rem] bg-[#1d1c20] border border-white/[0.08] rounded-lg p-4 sm:p-6 md:p-8 relative overflow-hidden',
+      props.class,
+    ]"
+  >
+    <h2 v-if="title" :class="['text-white text-lg mb-2', props.titleClass]">
```

Comment:
We can use named slot here, giving user more freedom for title and description. This will also remove the need for titleClass and descriptionClass (these will not provide intellisense in code editor)

Reply:
It seems like a better idea, will implement it